### PR TITLE
improve not found page

### LINKED ISSUE #164: Improve the Not found page to match other pages
https://monosnap.com/file/GRlrhi1LmS3JmP5IWRJtekeNRu0RjL

### PR DESCRIPTION
**Pull request by sudeeptarlekar**
This fixed #164 

@breezyboa _a please review
cc @rishimohan

### CODE REVIEW

File: app/javascript/Admin/Main.elm [hunk 1]
```diff
@@ -378,46 +379,70 @@ subscriptions model =
...
+                    , False
+                    )
 
                 Dashboard ->
-                    div [] [ text "Dashboard" ]
+                    ( div [] [ text "Dashboard" ], False )
 
                 _ ->
-                    div [] [ text "Not Found" ]
+                    ( div [] [ Errors.notFound ], True )
```
Comment (breezyboa) [inline app/javascript/Admin/Main.elm]: Returning a tuple for every page type is unnecessary and is slightly unclear.
why not create a separate function that wraps the Html of each page and call it for all pagetypes but handle the view for NotFound type separately

Also get rid of _ and address NotFound explicitly

**Comment by breezyboa** [comment]
@sudeeptarlekar _a

**Comment by sudeeptarlekar** [comment]
@breezyboa _a please take a look

**Comment by breezyboa** [comment]
Looks good
@rishimohan _a please review styles

**Pull request merged**